controllers/Article.js

```diff
@@ -0,0 +1,92 @@
+const mongoose = require('mongoose');
+const Article = require('../models/Article')
+
+
+exports.postArticle =async(req,res,next) =>{
```

Comment:
instead of exporting your functions at the same like of declaration, try to create functions and export them at the end of the file